src/borg/helpers/msgpack.py

```diff
@@ -84,7 +85,7 @@ def __init__(
         use_single_float=False,
         autoreset=True,
         use_bin_type=USE_BIN_TYPE,
-        strict_types=False
+        strict_types=False,
```

Comment:
Changed this after GitHub Actions complained, https://github.com/pgerber/borg/actions/runs/3308624366/jobs/5461145547#step:3:32.

> did you install black via pip install -r requirements.d/codestyle.txt?

Not that I remember.

Reply:
I could move it to a different commit to keep the patch cleaner. Should I?

Reply:
Hmm, guess I need to check what version it runs on gh actions. Usually it should not complain about code you did not add or modify.

Reply:
https://github.com/borgbackup/borg/pull/7133/files

Reply:
^ guess that will avoid such surprises.

Reply:
please:
```
# update your local master from github upstream
# in your branch:
git rebase -i master
pip install -r requirements.d/codestyle.txt
black .
```
guess that should get the style into sync.

Reply:
can you do that?

Reply:
That didn't actually help. Black still complained but since I'm about to move my changes to `helpers.fs`, this shouldn't matter for this pull request.

Reply:
Maybe `pip install -Ur ...` or so is needed, so it will change the version of black even if it is already installed in some version. Or uninstall / reinstall black.